engine/src/sysspec.cpp

```diff
@@ -1006,7 +1006,10 @@ static bool MCS_getentries_callback(void *p_context, const MCSystemFolderEntry *
 	return true;
 }
 
-bool MCS_getentries(bool p_files, bool p_detailed, MCListRef& r_list)
+bool MCS_getentries(MCStringRef p_folder,
+                    bool p_files,
+                    bool p_detailed,
+                    MCListRef& r_list)
 {    
 	MCAutoListRef t_list;
     
```

Comment:
@peter-b I can't seem to comment on lines beyond the standard context that GitHub shows. However what I'm suggesting is: `if (!p_files && p_folder == nil)`